zip_extractor/requirements.txt

```diff
@@ -0,0 +1,2 @@
+zipfile37
```

Comment:
Please correct this

Reply:
It is the library name. Can I leave the library names as just "zipfile"

Reply:
But what about the version. You can use "pip freeze > requiements.txt" to check that

Reply:
I don't know that feature. If you know that can you update the readme file.

Reply:
ok, that's an inbuilt library. My bad